app/src/main/java/com/github/sdp/mediato/data/Database.java

```diff
@@ -214,4 +221,45 @@ public static CompletableFuture<String> addCollection(String username, Collectio
         return future;
     }
 
+    /**
+     * Method to follow a user
+     *
+     * @param myUsername       the current user's username
+     * @param usernameToFollow the username of the user to follow
+     */
+    public static void followUser(String myUsername, String usernameToFollow) {
+        CompletableFuture<String> future = new CompletableFuture<>();
+        database.getReference()
+                .child(USER_PATH + myUsername + FOLLOWING_PATH + usernameToFollow).setValue(true)
+                .addOnCompleteListener(task -> {
+                    System.out.println(myUsername + " is now following " + usernameToFollow);
+                });
+        database.getReference()
+                .child(USER_PATH + usernameToFollow + FOLLOWERS_PATH + myUsername).setValue(true)
+                .addOnCompleteListener(
+                        task -> {
+                            System.out.println(usernameToFollow + " is now followed by " + myUsername);
+                        });

```

Comment:
could those two pieces of code be used in a private method to avoid repetition ? It should be possible by giving it parameters FOLLOWING/FOLLOWER for ex.

Reply:
Thanks for the comment. I adressed this in commit [ef4f04b] by adding two helper methods, one for setting the username's value (true or false) in the follower's list and another one for the following's list. I think it's more convenient to keep the following/follower in separate methods.